packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -2,6 +2,8 @@
   @apply flex flex-col;
   --calcite-list-item-icon-color: theme("colors.brand");
   --calcite-list-item-spacing-indent: theme("spacing.4");
+  // uncomment this and it works with `forceFallback: true,`
+  // padding-block: 10px;
```

Comment:
Introducing a padding between items seems to fix the dragging when fallback is enabled. Need to figure out why dragging on a web component isnt triggering elements to swap unless there is space between them. cc @jcfranco